src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.TaskList;
+
+/**
+ * The Duke class that instantiates instances of duke.
+ *
+ * Duke is a ChatBot that performs different actions
+ * based on commands provided by user.
+ *
+ * CS2103T IP
+ * AY22/23 Semester 1
+ * @author Tan Jia Rong
+ */
+public class Duke {
+    // Location of save file
+    private static final String SAVE_LOCATION = "./data/data.txt";
```

Comment:
Good use of capitalisation for constants

Reply:
Thanks ❤️